### PR TITLE
FIX: OnScreenButton leaving stuck keys when disabled.

### PR DESCRIPTION
**Pull request by Rene-Damm**
Also contains a minor change to tag virtual devices created by OnScreenControls with "OnScreen" and to show device usage tags in the Input Debugger's device list.

### CODE REVIEW

File: Packages/com.unity.inputsystem/InputSystem/Editor/Debugger/InputDebuggerWindow.cs [hunk 1]
```diff
@@ -657,18 +657,22 @@ private void AddUser(TreeViewItem parent, InputUser user, ref int id)
...
+                    var displayName = device.name;
+                    if (device.usages.Count > 0)
+                        displayName += " (" + string.Join(",", device.usages) + ")";
+
                     var item = new DeviceItem
                     {
                         id = id++,
                         depth = parent.depth + 1,
-                        displayName = device.name,
+                        displayName = displayName,
```
Comment (karljj1) [inline Packages/com.unity.inputsystem/InputSystem/Editor/Debugger/InputDebuggerWindow.cs:678]: Is this correct? It looks like it is assigning the same value to itself?

Comment (karljj1) [reply Packages/com.unity.inputsystem/InputSystem/Editor/Debugger/InputDebuggerWindow.cs:678]: oh wait they just have the same name.

File: Packages/com.unity.inputsystem/InputSystem/Plugins/OnScreen/OnScreenButton.cs [hunk 2]
```diff
@@ -39,6 +51,8 @@ public void OnPointerDown(PointerEventData eventData)
         [SerializeField]
         private string m_ControlPath;
 
+        [NonSerialized] bool m_Pressed;
```
Comment (karljj1) [inline Packages/com.unity.inputsystem/InputSystem/Plugins/OnScreen/OnScreenButton.cs]: I dont think you need NonSerialized if the field is already private. Its the default behaviour.

**Review by karljj1 — APPROVED** [review]

File: Packages/com.unity.inputsystem/InputSystem/Plugins/OnScreen/OnScreenButton.cs (hunk 2 above)
Comment (Rene-Damm) [reply Packages/com.unity.inputsystem/InputSystem/Plugins/OnScreen/OnScreenButton.cs]: Ended up reworking the change which removed the field altogether.

**Pull request merged**